### DISCUSSION TITLE
Variant of `v-btn` within `v-app-bar` not configurable?

### DISCUSSION THREAD

**Discussion by Cyclodex** [vuetifyjs/vuetify#16466] [Q&A]
I was wondering why my buttons were not behaving normally, and I found out that they apply text variant because they are within v-app-bar.

The info is noted in the documentation:

> When a v-btn is used inside of v-toolbar and v-app-bar the default variant text is applied instead of elevated.

But why are the v-btn forced to be of variant "text" within v-app-bar?
Why is it not configurable? I tried it over

```
defaults: {
  global: {
    variant: "outlined",
  },
  VBtn: {
    variant: "outlined",
  },
  VAppBar: {
    variant: "outlined"
  }
},
```

Nothing of the above works and so we need to tell every `v-btn`, that it should not behave like a **text-variant**. I am wondering what's the reason behind this and if the documentation maybe can be more explicit about it.

Thanks !

**Comment by Picturk** [comment]
Hi Cyclodex, 

I'm trying to get a v-btn to render as a regular button in the app-bar too. Did you find a work around to get the buttons to render the way you want them to render? 

(You could apply the v-btn variant attribute in version 2)

P

**Comment by KaelWD** [comment]
```
VAppBar: {
  VBtn: { variant: 'outlined' }
}
```
